codeHF/config_tasks.sh

```diff
@@ -61,7 +61,7 @@ DOO2_CAND_DSTAR=0   # hf-candidate-creator-dstar
 # Selectors
 DOO2_SEL_D0=0       # hf-candidate-selector-d0
 DOO2_SEL_DS=0       # hf-candidate-selector-ds-to-k-k-pi
-DOO2_SEL_DPLUS=0    # hf-candidate-selector-dplus-to-pi-k-pi
+DOO2_SEL_DPLUS=1    # hf-candidate-selector-dplus-to-pi-k-pi
```

Comment:
1. When I run Run3Analysisvalidation with the default settings, it crashes because the D0 selector needs tables that are produced by `HfCandidateSelectorDplusToPiKPiExpressions` from `PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx`.
1b. D0 selector is disabled here, but later implicitly switched on because of default `APPLYCUTS_D0=1`.

ad. 1.:
Maybe one day `HfCandidateSelectorDplusToPiKPiExpressions` should be extracted from `candidateSelectorDplusToPiKPi.cxx` to make it a clearer dependency to the D0 selector as well?

ad. 1b.:
Maybe by default we could set `DOO2_SEL_D0=1`? The task is anyways switched on, and this would make it clearer. The fact that selector is included in spite of `DOO2_SEL_D0=0` is a bit confusing at first sight.
Would it make sense to change the logic to: 
1) attach the selector task only if both selector and the corresponding cut flags are true (`DOO2_SEL_D0=1 && APPLYCUTS_D0=1`)
2) error and warn the user if flags are mixed ("You need to enable the ... selector if you want the ... cut")
3) do not attach the selector task if both flags are false
? Or is it against your original design? Was it your intention to enable attaching the task with just the cut flag on, i.e., `APPLYCUTS_D0=1`?

Reply:
The crashing selector was fixed last Wednesday in https://github.com/AliceO2Group/O2Physics/pull/3872

Reply:
`DOO2_SEL_D0` and `APPLYCUTS_D0` are independent. The first one enables the selector workflow, the second one applies the candidate selection in the analysis task. Having `DOO2_SEL_DPLUS=1` is not needed, because the `config_task.sh` switches represent what the user explicitly asks for and the dependencies are enabled recursively through the database.

Reply:
The idea is: "Tell me what you want and let me do the rest." :-)

Reply:
"Having DOO2_SEL_D0=1 is not needed" --> I mean, doesn't it cause confusion then?

Reply:
What kind of confusion?

Reply:
It is just like running a wagon test on Hyperloop. You specify the dependencies in the wagon configuration and to run it, you just press a single button which runs all needed workflows together.

Reply:
ok